### ISSUE TITLE
Svelte 3

### ISSUE THREAD

**Issue by stefanomasini** [bsssshhhhhhh/svelte-data-grid#1]
Hi,

any plan to port this library to Svelte 3?

**Comment by BrianESimon** [comment]
Yes, I'll be working on a port to svelte 3 within the next few weeks.

**Comment by MintyMods** [comment]
I would also be interested in a V3 port of this component

**Comment by BrianESimon** [comment]
I'm working actively on a Svelte 3 port.

**Comment by arperyan** [comment]
Any update on this?

**Comment by thecodejack** [comment]
@bsssshhhhhhh any update...where are we on this? I can help you if needed...

**Comment by hangtwenty** [comment]
I volunteer to help as well, is there a branch yet? (Checked and did not see one, but I'd be glad to just grab it even if it's messy, and see what I can do)

**Comment by floer32** [comment]
I just saw in the README (on master) where it says this works with v3

😄
